src/components/input/input-theme.scss

```diff
@@ -37,6 +37,66 @@ md-input-container.md-THEME_NAME-theme {
         color: '{{foreground-2}}';
       }
     }
+  }
+  .md-input {
+    &[disabled],
+    [disabled] & {
+      border-bottom-color: transparent;
+      color: '{{foreground-3}}';
+      background-image: linear-gradient(to right, '{{foreground-3}}' 0%, '{{foreground-3}}' 33%, transparent 0%);
+      background-image: -ms-linear-gradient(left, transparent 0%, '{{foreground-3}}' 100%);
+    }
+  }
+}
+
+md-toolbar.md-THEME_NAME-theme:not(.md-menu-toolbar) md-input-container {
```

Comment:
this should not be in the `input-theme.scss`. It should be in the `toolbar-theme.scss` right ?